### DISCUSSION TITLE
v0.2.4 - Dynamic number spacing while editing

### DISCUSSION THREAD

**Discussion by c3d** [c3d/db48x#102] [Announcements]
Bug fixes:

- CHS removes negative sign instead of putting a + sign
- Fix crash when applying decimal functions to negative values (#101)
- Use default formatting when saving calculator state
- Repair rendering of FIX, SCI and ENG on the stack (#100)
- Fixes the selection of the editor's displayed rows and columns (#98, #99)
- Remove premature "too many digits" error message for decimal128 (#97)
- Do not terminate editing for self-inserting menus such as `#` (#96)
- Show B cursor for all base-36 digits (#95)
- Defensive coding against malformed UTF8 text rendering (#101)

Features:

- Show thousand separators while editing numnbers
- Render based numbers with a numerical postfix base indicator (#95)
- Parse numbers in arbitrary base, e.g. `3#212` (#95)

**Full Changelog**: https://github.com/c3d/DB48X-on-DM42/compare/v0.2.3...v0.2.4

<hr /><em>This discussion was created from the release <a href='https://github.com/c3d/DB48X-on-DM42/releases/tag/v0.2.4'>v0.2.4 - Dynamic number spacing while editing</a>.</em>

**Comment by balta1980** [comment]
Thank you very much, great great calculator you are doing.
